payloads/wallet.ts

```diff
@@ -0,0 +1,122 @@
+import { Psp } from "../generated/definitions/pagopa/Psp";
+import { validatePayload } from "../src/utils/validator";
+import { SessionResponse } from "../generated/definitions/pagopa/SessionResponse";
+import { WalletResponse } from "../generated/definitions/pagopa/WalletResponse";
+import { TransactionResponse } from "../generated/definitions/pagopa/TransactionResponse";
+import { getRandomIntId } from "../src/utils/id";
+
+export const sessionToken: SessionResponse = {
+    data: {
+        sessionToken: '3m3Q2h6e8T5w9t3W8b8y1F4t2m6Q9b8d9N6h1f2H2u0g6E7m9d9E3g7w3T3b5a7I4c4h6U4n2b3Z4p3j8D6p4a5G1c4a8K3o0v8P7e0j6R5i1y2J6d0c7N9i6m0U3j9z'
+    }
+}
+
+export const getWallets = (): WalletResponse => {
+    const validCreditCard: { [key: string]: any } = {
+        id: 1464,
+        holder: "Mario Rossi",
+        pan: "************0111",
+        expireMonth: "05",
+        expireYear: "22",
+        brandLogo:
+          "https://wisp2.pagopa.gov.it/wallet/assets/img/creditcard/carta_mc.png",
+        flag3dsVerified: true
+    };
+  
+    const validAmount: { [key: string]: any } = {
+        currency: "EUR",
+        amount: 1000,
+        decimalDigits: 2
+    };
+  
+    const validPsp: { [key: string]: any } = {
+        id: 43188,
+        idPsp: "idPsp1",
+        businessName: "WHITE bank",
+        paymentType: "CP",
+        idIntermediary: "idIntermediario1",
+        idChannel: "idCanale14",
+        logoPSP:
+          "https://wisp2.pagopa.gov.it/pp-restapi/v2/resources/psp/1578833",
+        serviceLogo:
+          "https://wisp2.pagopa.gov.it/pp-restapi/v2/resources/service/1578833",
+        serviceName: "nomeServizio 10 white",
+        fixedCost: validAmount,
+        appChannel: false,
+        tags: ["MAESTRO"],
+        serviceDescription: "DESCRIZIONE servizio: CP mod1",
+        serviceAvailability: "DISPONIBILITA servizio 24/7",
+        paymentModel: 1,
+        flagStamp: true,
+        idCard: 91,
+        lingua: "IT"
+    };
+
+    enum TypeEnum {
+        "CREDIT_CARD" = "CREDIT_CARD",
+        "BANK_ACCOUNT" = "BANK_ACCOUNT",
+        "EXTERNAL_PS" = "EXTERNAL_PS"
+    };
+  
+    const WalletCard: Wallet = {
+        idWallet: 12345,
+        type: TypeEnum.CREDIT_CARD,
+        favourite: false,
+        creditCard: validCreditCard as CreditCard,
+        psp: validPsp as Psp,
+        idPsp: validPsp.id,
+        pspEditable: true,
+        lastUsage: new Date("2018-08-07T15:50:08Z")
+    };
+  
+    // It is displayed as card!
+    const WalletBank: Wallet = {
+        idWallet: 67890,
+        type: TypeEnum.CREDIT_CARD,
+        creditCard: validCreditCard as CreditCard,
+        psp: validPsp as Psp,
+        idPsp: validPsp.id,
+        pspEditable: true,
+        lastUsage: new Date("2018-08-07T15:50:08Z")
+    };
+  
+    return {
+        data: [WalletBank, WalletCard]
+    } ;   
+}
+
+export const getTransactions = (amount: number): TransactionResponse => {
```

Comment:
```suggestion
export const getTransactions = (count: number): TransactionResponse => {
```
I guess count is more appropriate

Reply:
updated within https://github.com/pagopa/io-dev-api-server/pull/1/commits/9a7b3c5dc219c65fbcbfb87f2e1b29ba2030f538